common_sensor_launch/launch/nebula_node_container.launch.py

```diff
@@ -333,6 +406,9 @@ def add_launch_arg(name: str, default_value=None, description=None):
     add_launch_arg("use_multithread", "False", "use multithread")
     add_launch_arg("use_intra_process", "False", "use ROS 2 component container communication")
     add_launch_arg("lidar_container_name", "nebula_node_container")
+    add_launch_arg("pointcloud_container_name", "pointcloud_container")
+    add_launch_arg("use_pointcloud_container", "False")
+    add_launch_arg("use_cuda_preprocessor", "False")
     add_launch_arg("ptp_profile", "1588v2")
```

Comment:
`lidar_container_name`, `pointcloud_container_name` and `use_pointcloud_container` could be confusing for users, how about for example `container_name` and `use_shared_container`, with the appropriate `description=`?

Reply:
Also, are users always expected to use a shared container when using CUDA?

Reply:
Addressed in [411c39f](https://github.com/tier4/aip_launcher/pull/348/commits/411c39feaff2c906c06e60c194489a16a9adeeb2), 
315d2a4, and [f79d487](https://github.com/tier4/aip_launcher/pull/348/commits/f79d4875c315725de3562029a4904b548ac9b547).

I made it so that it was required (otherwise performance will not be good and I prefer it not working than working bad without the user realizing)